File: raspberryPi3/wiringPi/c/src/blink.c
/**
** Program: blink.c
** Description: Blink led using 
** 				wiringPi library
** Author: Aldo Nunez
**/

#include <stdio.h>
#include <wiringPi.h>

int
main ( void )
{
	/* Port number */
	const int PIN = 7;

	/* Delay in milliseconds */
    const int T = 50; 	
	
    if ( wiringPiSetup () )
	{
		fprintf ( stderr,  "Failing to setup wiringPi\n" );
		return 1;
	}

	/* set the port as output */
    pinMode ( PIN, OUTPUT );

	puts ( "blinking" );
    while ( 1 )
    {
		/* Turn on */
        digitalWrite ( PIN, HIGH );
		/* waiting */
        delay ( T );

		/* Turn off */
        digitalWrite ( PIN, LOW );
		/* waiting */
        delay ( T );
    }

    return 0;
}

File: raspberryPi3/wiringPi/python/blink.py
'''
Program: blink.py
Description: Blinking led
			using wiringPi library
Author: Aldo Nunez
'''
# Import GPIO module
import wiringpi
import time 

def main ():
	# port number
	PIN = 7;
	# time delay in milliseconds
	t = 0.05 	# 0.05 ~ 50 ms

	wiringpi.wiringPiSetup()
 	wiringpi.pinMode ( PIN, 1 )

	while True:
		wiringpi.digitalWrite ( PIN, 1)
		time.sleep ( t )
		wiringpi.digitalWrite ( PIN, 0) 
		time.sleep (0.5)

if __name__  == "__main__":
	main ()
